src/content/post/2025-06-11-version-6-6-6.mdx

```diff
@@ -0,0 +1,61 @@
+---
+publishDate: 2025-06-11T17:00:00Z
+author: ildyria
+title: 'Version 6.6.6 and what is next?'
+excerpt: Why is version 6.6.6 special? What is next on the roadmap for Lychee?
+image: https://images.unsplash.com/photo-1660262848906-319f55766d2b?q=80&w=3474&auto=format&fit=crop&ixlib=rb-4.1.0&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D
+category: Release Notes
+tags:
+  - lychee
+  - blabla
+  - Supporter Edition
+  - v6
+---
+
+Five days ago, we looked back on what we have accomplished over the past six months.
+
+#### What is special about version 6.6.6?
+
+Today, we are happy to announce the release of Lychee [version 6.6.6](https://github.com/LycheeOrg/Lychee/releases/tag/v6.6.6). An ill-fated version number which marks a brutal change in the way Lychee binds photos to albums.
+We are moving from "*a photo can only be in a single album*" to "*a photo can be in multiple albums*".
+
+To "*render unto Caesar*", [Matthias Nagel](https://github.com/nagmat84) was the first to suggest it, and I was not convinced that it was a good idea.
+It has always been a feature that I thought was not needed, and I was afraid that it would complicate the codebase.
+Now having worked on creating a timeline view, I can see how useful it is to have a photo in multiple albums: it avoids duplication.
+
+While invisible to the user, this architecture change has a big impact. By having the relation a photo has a unique album,
+it makes it possible to do a file mapping between the photos/albums in Lychee and the files/folders on the backend.
+While this is not implemented, it could be seen in the future to be made possible as this has been a long-standing complaint from some of our users.
```

Comment:
```suggestion
While this is not currently implemented, it could be in the future as this has been a long-standing complaint from some of our users.
```